mw-webapp/src/component/image/Image.module.scss

```diff
@@ -1,3 +1,15 @@
+.imageWrapperFullScreen {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
Don't enough only top and left for position image?

Reply:
@Ekaterina1994 changed